tasks.py

```diff
@@ -135,23 +156,23 @@ def generate_template_content(template_params):
     if 'replacements' not in template_params:
         raise KeyError('replacement values are required for email template')
 
-    with urllib.request.urlopen(template_params['url']) as conn:
-        template_content = conn.read()
-        if not isinstance(template_content, str):
-            template_content = template_content.decode("utf-8")
-        template = Template(template_content)
-        html_content = template.render(template_params['replacements'])
-
-        result.append({
-            "type": mimetypes.types_map['.html'],
-            "value": html_content
-        })
-
-        soup = BeautifulSoup(html_content, features="html.parser")
-        result.append({
-            "type": mimetypes.types_map['.txt'],
-            "value": soup.get_text()
-        })
+    index = template_params['url'].rfind('/')
+    path = template_params['url'][:index+1]
+    filename = template_params['url'][index+1:]
```

Comment:
maybe we can use something like https://docs.python.org/3/library/urllib.parse.html to parse URLs?

Reply:
i wanted to use urlparse, but doesn't provide the filename and path as separate values.

```
urlparse('//www.cwi.nl:80/%7Eguido/Python.html')
ParseResult(scheme='', netloc='www.cwi.nl:80', path='/%7Eguido/Python.html', params='', query='', fragment='')
```